sql/create.go

```diff
@@ -151,8 +151,27 @@ func (p *planner) CreateTable(n *parser.CreateTable) (planNode, *roachpb.Error)
 		return nil, pErr
 	}
 
-	if pErr := p.createDescriptor(tableKey{dbDesc.ID, n.Table.Table()}, &desc, n.IfNotExists); pErr != nil {
+	created, pErr := p.createDescriptor(tableKey{dbDesc.ID, n.Table.Table()}, &desc, n.IfNotExists)
+	if pErr != nil {
 		return nil, pErr
 	}
+
+	if created {
```

Comment:
I would argue that it is the attempt to create a table that should be logged and the result status could be complete success or any failure, including table exists and IF NOT EXISTS is not specified. I would also argue for a log column to indicate if an operation is essentially ignored such as the case of an existing table and IF NOT EXISTS.

So, the question is whether then intention of the table is to record attempts to perform an operation or only whether some change in system state/schema actually occurs.

How about the case where the user attempts to create a table without sufficient privileges? I would argue that such an attempt should be logged as a fairly serious event.